pkg/skaffold/runner/v3/generate_pipeline.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
not completely sure we'll be supporting this feature in the future, might be best to just leave it out of newer versions?
wdyt @tejal29 @nkubala ?

Reply:
+1

I'd vote for removing the generate-pipeline from v3, or at most p2. 
For the v2 main stream, skaffold+ tekton seems to be a nice combination. the tekton also use skaffold in their examples see [here](https://github.com/tektoncd/pipeline/blob/main/docs/tutorial.md#running-your-task)

Reply:
hmm i can see that generated yaml would change. We could re-implement what we have as P2.

We should also instrument its usage.